src/lib/mac-address.ts

```diff
@@ -0,0 +1,53 @@
+import * as _ from 'lodash';
+import * as fs from 'mz/fs';
```

Comment:
```suggestion
import { promises as fs } from 'fs';
```

We're slowly moving to this for everything in the supervisor

Reply:
this will break `fs.exists()` which is used below... Happy to change this, but would you rather have 2 imports, or import it like this?